### PR TITLE
Create agent-policies.asciidoc

### PR DESCRIPTION
**Pull request by EricDavisX**
This resolves 
https://github.com/elastic/observability-docs/issues/271
https://github.com/elastic/observability-docs/issues/395
https://github.com/elastic/observability-docs/issues/394

It is likely in need of attention from someone who actually knows the docs system - but hopefully all the copy / paste of the info from those 3 tickets and a head start on {kib} and {fleet} and {agent} and {policy} and {policies} variables will help.  I expect we'll want to do the same for {Elastic Package Registry} and more, but I didn't have as much a frame of reference for that so... I left it as regular old text.

In 271 Dede cited a preference to use a 'single longer' topic flow, but if this is in any way better off to be broken up, I'm all for it.  so long as it gets added for our uses!

### CODE REVIEW

**Comment by EricDavisX** [comment]
Dede and I spoke and are hoping to get someone with more writing focus to take it over and better it for us! :)

**Comment by bmorelli25** [comment]
Hey, @EricDavisX. Thanks for opening this PR!

> hoping to get someone with more writing focus

I spoke with DeDe this morning, and I'd like to help out ✋. I'm not sure how do you and DeDe typically work together on PRs like this–is it OK if I push directly to your branch?

**Comment by dedemorton** [comment]
@bmorelli25 Sometimes I will create a branch based on the PR branch and open my own PR so that there is less noise. That way, the original contributor gets credit for their work, but I can push with abandon. But of course, it's totally up to you two. Thanks SO much for doing this work!

**Comment by bmorelli25** [comment]
👋 @EricDavisX and @elastic/obs-docs. I pushed some changes to this PR in 0263e8e. The high-level flow of the content is roughly the same, but I've moved away from long paragraph topics, to shorter, itemized lists. I started to add screenshots but realized it would probably be better to open this up to reviews before I went too far down the screenshot rabbit hole.

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 1]
```diff
@@ -1,214 +1,215 @@
...
+A {policy} is a collection of inputs and settings that defines the data to be collected
+by {agent}. Each {agent} can only be enrolled in a single {policy}.
 
-[discrete]
-[[policy-use]]
-== How to Use {policy}
+Within an {agent} {policy} is a set of individual integration policies.
+These integration policies define the settings for each input type.
+The available settings in an integration relate to the version of
+the integration in use–newer functionality can be exposed in this manner over time.
```
Comment (EricDavisX) [inline docs/en/ingest-management/agent-policies.asciidoc:17]: looking at 'use-newer' I feel like maybe a ; would work better?  It confused me when I read it.

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 2]
```diff
@@ -1,214 +1,215 @@
...
+and agent monitoring status can be changed as necessary:
+
+. In {fleet}, click *Policies*.
+Click on the name of the policy you want to edit or delete.
+
+. Click the *Settings* tab, make changes, and click *Save changes*
++
+Alternatively, click *Delete policy* to delete the policy.
+Existing data will not be deleted.
+Any agents assigned to a policy must be unenrolled before a policy can be deleted.
```
Comment (EricDavisX) [inline docs/en/ingest-management/agent-policies.asciidoc:153]: we could make it more clear here with something like ...  "unenrolled, or simply assigned to a different policy, before a policy..."

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 3]
```diff
@@ -1,214 +1,215 @@
...
+it will show up in the Integrations listing in {kib} after a restart (like during a stack upgrade).
+// to do: link to the Fleet API docs
+A refresh can also be triggered manually, with the Fleet API.
 
-[discrete]
-[[integration-update-steps1]]
-== How to update an Integration version in a {policy}
-
-To check if a new Integration exists, follow the {policy} management documentation steps above to view
-a {policy} with the Integration whose version is to be compared against the available Integrations
```
Comment (EricDavisX) [inline docs/en/ingest-management/agent-policies.asciidoc]: lol, this section is really rambly.  thanks for just taking it out.  i will have to look to see what we have in 7.13 and what is coming in terms of 'indicators' to help users know they need to update an integration.  it is an area we are improving, so the exact state is tbd until I go look at code and prs coming.

**Review by EricDavisX — COMMENTED** [review]
looks great - thanks for the edits.  the nuances on integration update needs is TBD as I noted, the team is improving this but it may not come until next release.  Also in next release, the 'Integrations' tab in Fleet will be moving to a top-level application in Kibana so we'll need to figure out how to link to that and adjust each of these references to it.  : /

**Comment by EricDavisX** [comment]
and we can track this issue issue for updates on if we'll make changes:  https://github.com/elastic/kibana/issues/92612

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+Any agents assigned to a policy must be unenrolled before a policy can be deleted.
+
+[discrete]
+[[integration-updates]]
+== Integration updates
+
+Elastic releases integration updates periodically.
+Integrations are delivered to the Elastic Stack, via the online Elastic Package Registry
+and the {kib} Elastic Package Manager. When a new Integration is released by Elastic,
+it will show up in the Integrations listing in {kib} after a restart (like during a stack upgrade).
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:162]:
```suggestion
Through the online Elastic Package Registry and the {kib} Elastic Package Manager, Integrations are delivered to the {stack}. When Elastic releases a new Integration, it shows up in the Integrations listing in {kib} after a restart (like during a stack upgrade).
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+// to do: link to the Fleet API docs
+A refresh can also be triggered manually, with the Fleet API.
+
+NOTE: The latest, shipped Elastic Package registry version of an integration is the only version
+that can be integrated into {agent} policies. Previously installed versions will continue to work.
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:167]:
```suggestion
NOTE: The latest Elastic Package registry version of an Integration is the only version
integrated into {agent} policies. Previously installed versions will continue to work.
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+[[update-an-integration]]
+=== Update an integration
+
+. In {fleet}, click *Integrations*.
+Search for and select the integration you'd like to update.
+
+. If an update is available, click *Update*.
++
+Because updates can change the behavior of hosts,
+they must be explicitly request in the {fleet} app.
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:179]:
```suggestion
they must be explicitly requested in the {fleet} app.
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+
+. <<create-a-policy,Create a new policy>>.
+
+. <<add-integration,Add the integration to the policy>>.
+The newer version will automatically be used
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:184]:
```suggestion
The newer version is automatically used
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+
+. <<apply-a-policy,Apply the policy>> to an {agent}.
++
+TIP: In larger deployments, integration updates should be tested on a sample {agent}
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:188]:
```suggestion
TIP: In larger deployments, you should test integration updates on a sample {agent}
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+
+. Click *Add integration* and re-add the freshly deleted integration.
+The updated version will be used and applied to all {agent}s.
+
+. Repeat this process for each policy with the out-of-date integration.
+
+NOTE: In some instances, for example, when there are hundreds or thousands of different {agent}s and
+policies that need to be updated, this upgrade path is not feasible.
+In this case, update one policy and use the <<copy-policy>> action to apply the updated policy versions to additional policies.
+The downside of this method is losing
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:214]:
```suggestion
This method's downside is losing
```

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+the granularity of assessing the individual Integration version changes individually across {policies}.
```
Comment (EamonnTP) [inline docs/en/ingest-management/agent-policies.asciidoc:215]: Should the attribute be {policy}'s? Looks like the attribute could be used in a few other sections.

**Review by EamonnTP — COMMENTED** [review]
@bmorelli25 Just a few minor suggestions. Please feel free to take or leave them. Otherwise, it looks great!

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+in the {fleet} UI.
+
+[discrete]
+[[policy-benefits]]
+== Policy benefits
+
+{agent} policies allow you to indicate an intended logical grouping of inputs aimed for a particular set of hosts.
+This allows you maintain flexibility in large-scale deployments by
+easily testing changes before rolling them out,
+and providing a way to group and manage larger swaths of your infrastructure landscape.
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/en/ingest-management/agent-policies.asciidoc:32]: niiiiice

File: docs/en/ingest-management/agent-policies.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/en/ingest-management/agent-policies.asciidoc:17]:
```suggestion
the integration in use; newer functionality can be exposed in this manner over time.
```

File: docs/en/ingest-management/agent-policies.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/en/ingest-management/agent-policies.asciidoc:152]:
```suggestion
Any agents assigned to a policy must be unenrolled or assigned to a different policy before a policy can be deleted.
```

File: docs/en/ingest-management/agent-policies.asciidoc (hunk 10 above)
Comment (bmorelli25) [reply docs/en/ingest-management/agent-policies.asciidoc:212]:
```suggestion
the granularity of assessing the individual Integration version changes individually across policies.
```

**Comment by bmorelli25** [comment]
> looks great - thanks for the edits. the nuances on integration update needs is TBD as I noted, the team is improving this but it may not come until next release.

👍 Thanks for providing elastic/kibana#92612. I'll open a documentation issue so we don't forget to update these docs for 7.13.

> Also in next release, the 'Integrations' tab in Fleet will be moving to a top-level application in Kibana so we'll need to figure out how to link to that and adjust each of these references to it. : /

😨 Good to know! I'll open an issue for this as well

---

Thank you @EricDavisX for providing original content and a review and thanks @EamonnTP for the review! I have a few more commits to push based on reviews, and then we'll get this merged today 🍻

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+This will save the integration policy as a part of the larger {agent} {policy}.
+{fleet} will distribute this new {policy} to all {agent}s that are enrolled with it.
+
+After the {policy} has finished applying, the selected integration will be running on the host
+and communicating with the {agent}!
+
+[discrete]
+[[apply-a-policy]]
+== Apply a policy
+
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply docs/en/ingest-management/agent-policies.asciidoc:89]: Will do

**Review by bmorelli25 — APPROVED** [review]
LGTM

**Comment by EricDavisX** [comment]
#WeDidIt #MissionAccomplished - great team work, so much good stuff for our users!

**Pull request merged**